components/perl/Razor2-Client-Agent/Makefile

```diff
@@ -0,0 +1,37 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# This file was automatically generated using the following command:
+#   $WS_TOOLS/perl-integrate-module Razor2::Client::Agent
+#
+
+BUILD_STYLE = makemaker
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_PERL_MODULE =		Razor2::Client::Agent
+HUMAN_VERSION =			2.86
+COMPONENT_REVISION =		1
```

Comment:
This line should not be there.  I'm curious why it appeared here.  Did you run  the `perl-integrate-module` script twice and then squashed both changesets?  Or maybe there is a bug in the `perl-integrate-module` script...

Anyway, this is harmless, so no problem to leave it here.

Reply:
Shall I merge this as is or wait for a correction?

Reply:
> Shall I merge this as is or wait for a correction?

Correction is not needed, but I'd like to know the answer to my question.
So feel free to merge.

Thanks.